test/topology_experimental_raft/test_tombstone_gc.py

```diff
@@ -53,24 +53,43 @@ async def test_default_tombstone_gc_does_not_override(manager: ManagerClient, rf
 
 
 @pytest.mark.asyncio
-@pytest.mark.xfail(reason="issue #15607")
```

Comment:
I see you're modifying the test in the same commit that you're un-xfailing it. Can't it have its final form when you introduce it? Otherwise, the whole "add test, making it xfail -> fix test" idea kind of falls apart (if you need to rewrite it anyway; the final form does something different than the initial xfailing form)

Reply:
done, added the whole test to the test commit.